### DISCUSSION TITLE
Outdated Hash File (404 Error) for Older Versions

### DISCUSSION THREAD

**Discussion by Bouts2019** [k3s-io/k3s#7665] [Q&A] [answered]
I noticed that the hash file for older versions of K3s is returning a 404 error. Specifically, when trying to access the following URL:
https://github.com/k3s-io/k3s/releases/download/v1.21.14/sha256sum-amd64.txt

It seems that the hash file for version 1.21.14 is not available, causing inconvenience for users who rely on it for verifying the integrity of their downloads.

Could the maintainers please address this issue and provide the correct hash file for version 1.21.14? It would greatly help ensure the security and trustworthiness of the K3s installation process.

Thank you for your attention to this matter.

**Comment by brandond** [comment, marked as answer]
`v1.21.14` is not a valid k3s release version. Are you looking for `v1.21.14+k3s1`?
